packages/augur-ui/src/modules/trading/components/form/form.styles.less

```diff
@@ -169,7 +171,7 @@
 .TradingForm__input {
   display: flex;
   flex-grow: 1;
-  padding: 0.4375rem 1rem;
+  padding: 0.4375rem @size-16;
```

Comment:
I believe this would be `@size-7` which may or may not already exist, but if it doesn't add it and replace that `0.4375rem` amount